templates/commands/render/flags.go

```diff
@@ -91,6 +94,14 @@ func (r *RenderFlags) Register(set *cli.FlagSet) {
 		Usage:   "The key=val pairs of template values; may be repeated.",
 	})
 
+	f.StringVar(&cli.StringVar{
```

Comment:
This can wait for a future PR, but I wonder if we should accept multiple --input-file arguments, and they would be merged?

Reply:
No strong opinion. The customer workaround would be to concatenate the input files into a single tmp file (i.e. `cat file1.yaml file2.yaml > /tmp/inputs.yaml`). However if it turns out to be a common use case, I'm willing to go ahead and support it.